WordPress/src/main/java/org/wordpress/android/WordPressDB.java

```diff
@@ -165,7 +164,7 @@
     //add boolean to posts to check uploaded posts that have local changes
     private static final String ADD_LOCAL_POST_CHANGES = "alter table posts add isLocalChange boolean default 0";
 
-    // Add boolean to POSTS to track posts currently being uploaded
+    // Add boolean to POSTS to track posts currently being uploaded - no longer used as of v4.5
     private static final String ADD_IS_UPLOADING = "alter table posts add isUploading boolean default 0";
```

Comment:
Makes sense - added in c26474a. We devs should keep in mind, though, that switching from this branch to an earlier one may cause a crash due to the missing column.